src/rhsmlib/facts/all.py

```diff
@@ -28,7 +29,8 @@ def __init__(self):
             hwprobe.HardwareCollector(),
             custom.CustomFactsCollector(),
             insights.InsightsCollector(),
-            kpatch.KPatchCollector()
+            kpatch.KPatchCollector(),
+            pkg_arches.PkgArchesCollector(),
```

Comment:
Module `pkg_arches` does not include any class called `PkgArchesCollector` it only includes class `SupportedArchesCollector`. It breaks more than 1000 unit tests.